src/index.ts

```diff
@@ -99,6 +100,7 @@ export {
 export { BatchType } from './bulk/common';
 export { AutoEncryptionLoggerLevel } from './client-side-encryption/auto_encrypter';
 export { GSSAPICanonicalizationValue } from './cmap/auth/gssapi';
+export { TokenCache, TokenEntry } from './cmap/auth/mongodb_oidc/token_cache';
```

Comment:
Since these are internal you can just export them as `type` we do below line 161 to make api-extractor happy. They will still be runtime accessible in test files.

Reply:
Changed to type export.